packages/bitcore-node/src/models/coin.ts

```diff
@@ -67,19 +68,15 @@ class CoinModel extends BaseModel<ICoin> {
 
   async getBalance(params: { query: any }): Promise<{ confirmed: number, unconfirmed: number, balance: number }> {
```

Comment:
So after this change, this is really just the same as CoinStorage.collection.aggregate